### ISSUE TITLE
Implement Pre-mine

### ISSUE THREAD

**Issue by kingsleyh** [SushiChain/SushiChain#277]
add an option to `sushid` to start up the node with a genesis block containing addresses and amounts. Using the `--pre-mine=premine.yml` command the node will read a yml file to generate the data.

example premine.yml:

```
addresses:
  - address: VDA2NjU5N2JlNDA3ZDk5Nzg4MGY2NjY5YjhhOTUwZTE2M2VmNjM5OWM2M2EyMWQz
    amount: 500
  - address: VDAyMzEwODI2NmE1MWJiYTAxOTA2YjE0NzRjYTRjYjllYTk0ZDZhYmJhZGU3MmIz
    amount: 9000
```

**Comment by watzon** [comment]
Beautiful

**Comment by kingsleyh** [comment]
Implemented by #282
